src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -37,8 +37,8 @@ export class IntroductionPage extends React.Component {
     const ombInfo = (
       <OMBInfoShared
```

Comment:
You can replace this with the [`va-omb-info` web component](https://design.va.gov/storybook/?path=/docs/components-va-omb-info--default)